docs/troubleshooting_guide.md

```diff
@@ -0,0 +1,358 @@
+### Troubleshooting Guide:
+This guide provides a searchable list of errors for a possible match/solution.
+
+
+#### 1. Missing `dg_name` in `cluster_config.json` manifestation:
+* If this is the error text:
+
+```bash
+fatal: [racnode1.orcl]: FAILED! => {"msg": "'ansible.vars.hostvars.HostVarsVars object' has no attribute 'dg_name'"}
+```
+
+* then this could possibly be the place to check:
+
+```bash
+✔ ~/git-clone-holder/bms-toolkit [master L|✚ 3…3] 
+00:52 $ cat cluster_config.json 
+[
+  {
+    "scan_name": "dec-test-scan-cluster",
+    "scan_port": "1521",
+    "cluster_name": "dec-test-cluster",
+    "cluster_domain": "dec-test-home",
+    "public_net": "dec-test-public_interface",
+    "private_net": "dec-test-private",
+    "scan_ip1": "172.16.30.20",
+    "scan_ip2": "172.16.30.21",
+    "scan_ip3": "172.16.30.22",
+    "dg_name": "DATA",<======================================== this parameter was missing in this file's this line
+    "nodes": [
+      {  "node_name": "racnode1.orcl",
+         "host_ip": "172.16.30.1",
+         "vip_name": "racnode1-vip.orcl",
+         "vip_ip": "172.16.30.11"
+      },
+      {  "node_name": "racnode2.orcl",
+         "host_ip": "172.16.30.2",
+         "vip_name": "racnode2-vip.orcl",
+         "vip_ip": "172.16.30.12"
+      }
+    ]
+  }
+]
+
+```
+
+#### 2. Error manifestation with respect to an invalid disk group name:
+* For following error:
+
+```bash
```

Comment:
Again not bash here;  not sure if there's an appropriate syntax-highlighting language for this.

Reply:
Removed